
#LENGTH SCALE FACTOR OF 2 APPLIED

link1_length: 0.088

link2_length: 0.168

link3_length: 0.28

body_length: 0.30

body_width: 0.30

body_height: 0.14
